product-scenarios/scenarios-commons/src/main/java/org/wso2/carbon/esb/scenario/test/common/ScenarioConstants.java

```diff
@@ -37,6 +37,19 @@
     public static final String VERSION_640 = "EI-6.4.0";
     public static final String VERSION_650_SNAPSHOT = "EI-6.5.0-SNAPSHOT";
     public static final String BASIC_JSON_MESSAGE = "{ \"name\":\"John\", \"age\":30, \"car\":null}";
+    public static final String SOAP_REUEST_1_6_3 = "<?xml version=\"1.0\" encoding=\"UTF-8\"?>\n"
```

Comment:
We should not add scenario specific constants here. If this is common placeOrder sample request for SimpleStockQuote service, then fine. But we should change the name to a common name 
Suggested name: SIMPLE_QUOTE_PLACEORDER

Reply:
This is done.